### PR TITLE
Revisions on solo staking guide

### PR DESCRIPTION
**Pull request by jholdstock**

### CODE REVIEW

File: docs/advanced/solo-proof-of-stake-voting.md [hunk 1]
```diff
@@ -39,19 +40,21 @@ Once your staking machine is set up you will need to install the Decred CLI tool
 
     `wget https://github.com/decred/decred-release/releases/download/v1.3.0/{dcrinstall-linux-amd64-v1.3.0,manifest-dcrinstall-v1.3.0.txt,manifest-dcrinstall-v1.3.0.txt.asc}`
 
-1. Verify the manifest.
+1. Verify the manifest. The output from this command should say "Good signature from Decred Release <release@decred.org>". Warnings about the key not being certified with a trusted signature can be ignore.
```
Comment (teknico) [inline docs/advanced/solo-proof-of-stake-voting.md]:
```suggestion
1. Verify the manifest. The output from this command should say "Good signature from Decred Release <release@decred.org>". Warnings about the key not being certified with a trusted signature can be ignored.
```

File: docs/advanced/solo-proof-of-stake-voting.md [hunk 2]
```diff
@@ -73,18 +76,16 @@ Once the wallet is created we can do a few things to make life easier.
...
+    `echo "tmux new -d -s dcrd 'dcrd'; tmux new -d -s dcrwallet 'dcrwallet --promptpass'; tmux attach -t dcrwallet" > ~/decred.sh;`
 
 1. Add the path to the Decred binaries to your `.profile`.
 
-    `"PATH=~/decred:$PATH" >> ~/.profile && source ~/.profile`
+    `echo "PATH=~/decred:$PATH" >> ~/.profile && source ~/.profile`
 
 1. Now make it executable with `chmod +x ~/decred.sh`
 
 Now you can start `dcrd`, `dcrwallet`, and unlock it just by running `./decred.sh`.
```
Comment (teknico) [inline docs/advanced/solo-proof-of-stake-voting.md]:
```suggestion
Now you can start `dcrd`, `dcrwallet`, and unlock the wallet just by running `./decred.sh`.
```

File: docs/advanced/solo-proof-of-stake-voting.md [hunk 3]
```diff
@@ -93,15 +94,15 @@ The next step will be to start buying tickets manually or using `ticketbuyer` in
 
 Now from your cool wallet you can purchase tickets using the following command once your wallet is unlocked:
 
-`dcrctl --wallet purchaseticket default 150 1 DsHotWalletAddressFromVPS 10`
+`dcrctl --wallet purchaseticket default 150 1 DsHotWalletAddressFromVotingWallet 10`
 
-Replacing `DsHotWalletAddressFromVPS` with your hot wallet voting address which you will generate when you set up your VPS for voting and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your VPS instances.
+Replacing `DsHotWalletAddressFromVotingWallet` with your hot wallet voting address which you will generate when you set up your remote servers for voting and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your hot wallets.
```
Comment (teknico) [inline docs/advanced/solo-proof-of-stake-voting.md]:
```suggestion
Replacing `DsHotWalletAddressFromVotingWallet` with your hot wallet voting address, which you will generate when you set up your remote servers for voting, and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your hot wallets.
```

File: docs/advanced/solo-proof-of-stake-voting.md [hunk 4]
```diff
@@ -93,15 +94,15 @@ The next step will be to start buying tickets manually or using `ticketbuyer` in
...
 
-If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf` where `DsHotWalletAddressFromVPS` is once again the address you generated on your voting VPS.
+If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf` where `DsHotWalletAddressFromVotingWallet` is once again the address you generated on your voting VPS.
```
Comment (teknico) [inline docs/advanced/solo-proof-of-stake-voting.md]:
```suggestion
If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf`, where `DsHotWalletAddressFromVotingWallet` is once again the address you generated on your voting VPS.
```

**Comment by jholdstock** [comment]
Thanks @teknico, I have incorporated all of your suggestions.

**Pull request merged**